docs/source/concept_models/vivarium_gates_bep/concept_model.rst

```diff
@@ -488,14 +488,46 @@ Apply the following intervention shifts according to interventions recieved in e
 8.0 Journal reviewer comments
 +++++++++++++++++++++++++++++
 
-We will do a sensitivity analysis on coverage for MMS, universal BEP and targeted BEP for a series of coverage points using one draw to explore relationship between the coverage proportion of the interventions and outcome (DALY's averted):
+.. note::
+
+  We have done a sensitivity analysis on coverage for MMS, universal BEP and targeted BEP for a series of coverage points using one draw to explore relationship between the coverage proportion of the interventions and outcome (DALY's averted):
+
+  - 0.4, 0.45, 0.5, 0.55, 0.6, 0.65, 0.7, 0.75, 0.8, 0.85, 0.9
+
+  Results are located here `/share/costeffectiveness/results/vivarium_gates_bep/sensitivity_analysis/`
 
-  - 0.4, 0.45, 0.5, 0.55, 0.6, .65, 0.7, 0.75, 0.8, 0.85, 0.9
+.. todo::
 
-Addtionally, we will update the following: 
+  Determine why increasing coverage from 0.75 to 0.8 results in fewer DALYs averted for Tanzania in sensitivity analysis
 
- - hopes hopes-and-dreams effect size of BEP 
- - maternal undernourishment effect size on birthweight (update lit review)
+For an updated run of the BEP simulation model for the repsonse to reviewers, we plan to include the following (for the full 100 simulation draws):
+
+ - updated hopes hopes-and-dreams effect size of BEP 
+ - updated maternal undernourishment effect size on birthweight (update lit review)
+ - sex stratification of outputs
+ - updated intervention coverage targets
  - SD calculation from confidence interval calculation should be (upper limit - lower limit)/2x1.96, and not 1.98.
-  
 
+.. todo::
+
+  Determine if SD calculation applies to simulation runs or researcher post-processing only.
```

Comment:
These should be wherever we used an input value with a 95%CI. The formula I gave @KjellSwedin used 1.98 instead of 1.96. Kjell, could you confirm that's was we used?

Reply:
We have a couple of places where 1.96 is used and one spot with 1.98

```
def twenty_percent_of_mean_variance(mean):
    ninety_five_percent_spread = .2 * mean
    std_dev = ninety_five_percent_spread / (2 * 1.96)
    return std_dev ** 2

def confidence_interval_std(upper, lower):
    ninety_five_percent_spread = (upper - lower)
    return ninety_five_percent_spread / (2 * 1.96)

CRUDE_BW_SHIFT_SD = (CRUDE_BW_SHIFT_UPPER - CRUDE_BW_SHIFT_LOWER) / (2 * 1.98)

```

Reply:
Great, thank you! I think we would like to update that 1.98 with 1.96 then.